server/post.go

```diff
@@ -137,3 +140,10 @@ func (p *Plugin) sendAnalytics() *model.AppError {
 
 	return nil
 }
+
+func channelLink(displayName string, link string) string {
```

Comment:
Maybe just pass `analyticsData` instead of two strings?

Reply:
:heavy_check_mark: